frontend/leaderboard/src/SheetsBinding.re

```diff
@@ -47,7 +57,20 @@ external get:
   (
     ~sheets: sheets,
     ~sheetsQuery: sheetsQuery,
-    (~error: string, ~res: res) => unit
+    (~error: Js.Nullable.t(string), ~res: res) => unit
   ) =>
   unit =
-  "get";
\ No newline at end of file
+  "get";
+
+type interface;
+
+[@bs.deriving abstract]
+type interfaceOptions = {input: in_channel};
+
+[@bs.module "readline"]
+external createInterface: interfaceOptions => interface = "createInterface";
+
+[@bs.send]
+external question: (interface, string, string => unit) => unit = "question";
+
+[@bs.send] external close: interface => unit = "close";
```

Comment:
Should these bindings be moved to a separate file as they are not part of Google Sheets?

Reply:
Up to you, you can also put them in a Readline module like
```
module Readline = {
...
}
```